### PR TITLE
Electron-builder error.

### PR DESCRIPTION
**Pull request by Prathik-Jain**
Downgraded electron-builder version to 20.14.7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +31,10 @@
     "node-notifier": "^5.1.2",
     "nojs": "^0.1.1"
   },
+  // change electron-builder version after https://github.com/electron-userland/electron-builder/issues/2958 is fixed.
```
Comment (ifedapoolarewaju) [inline package.json]: no need for comments, this often breaks the JSON parser :)

**Comment by Prathik-Jain** [comment]
Ready to merge?

**Comment by ifedapoolarewaju** [comment]
@Prathik-Jain please [see my comment](https://github.com/ifedapoolarewaju/igdm/pull/369#discussion_r197358162)

**Comment by Prathik-Jain** [comment]
Does this mean that the .AppImage file on the website is updated as well?

**Comment by Prathik-Jain** [comment]
@ifedapoolarewaju

**Comment by ifedapoolarewaju** [comment]
> Does this mean that the .AppImage file on the website is updated as well?

@Prathik-Jain no not yet. I'd have to release a new version before it becomes available. I'll try to do that soon.

**Comment by Prathik-Jain** [comment]
ETA?
@ifedapoolarewaju

**Comment by SimonVanacco** [comment]
@Prathik-Jain This did not solve the AppImage problem entirely: but i created a PR #374 that should fix it. Explained everything there.

**Pull request merged**